### PR TITLE
Add show frontend

### PR DESCRIPTION
**Pull request by SamuelSehnert**
Pull requested!

Currently, the page renders well and will only render the submit button upon valid entry of data.
Furthermore, the page renders differently depending on if it is passed data or not.

Will be used for editing media **AND** adding media

### CODE REVIEW

**Review by IgalTam — APPROVED** [review]
Good progress, but we'll have to sync up this file and the backend since the media attributes are slightly inconsistent. As one option, submitMedia() can be refactored to just do the API call and let the backend Media schema handle validation. Finally, "currentMinuities" seems to be typo'ed, which should be a quick fix.

**Comment by SamuelSehnert** [comment]
@IgalTam . Good points. I'll go ahead and fix the typo, and we can discuss the best way to mesh the object to the backend.

File: react-frontend/src/AddMedia.css [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    margin-top: 1em;
+    width: 90%;
+    height: 3em;
+}
+
+.show-info-parent {
+    display: flex;
+    margin-left: 5%;
+    margin-top: 1%;
+    /* width: 10%; */
```
Comment (mpicazocp) [inline react-frontend/src/AddMedia.css:20]: delete this

**Review by mpicazocp — APPROVED** [review]
Looks good to me, just commented a  style edit!

**Pull request merged**